src/types/misc.ts

```diff
@@ -190,11 +190,12 @@ export function isMatrix(x: any): x is Matrix<any> {
 }
 
 export interface ClipboardCell {
+  content?: string;
```

Comment:
Feels weird to have both `cell` and `content`. The first thing people reading this will ask themselves it what's the difference between `content` and `cell.content`. Same for `format`